chalice/app.py

```diff
@@ -28,6 +28,10 @@
 configure_lambda_logging()
 
 Config.set_config(BucketConfig.NORMAL)
+Config.BLOBSTORE_CONNECT_TIMEOUT = 5
```

Comment:
This is why I think the upper-case convention is useless, and actually misleading. Here you're reassigning to a constant. There are no constants in Python and we should pretend it does.

Anyhow, different PR.

Reply:
Yeah, I agree that this is kind of weird, but it's unclear to me what the solution is.